CLI/commands/common/common_functions.js

```diff
@@ -25,6 +26,38 @@ async function checkPermission(contractName, functionName, contractRegistry) {
   }
 };
 
+function getFinalOptions(options) {
+  const defaultOptions = {
```

Comment:
We could move this to constants.js

Reply:
This is missing, isn't it?

Reply:
I can't move it 'cos the constants were defined before Issuer and defaultGasPrice.

Reply:
Makes sense